src/main/java/com/wavefront/sdk/common/clients/WavefrontMultiClient.java

```diff
@@ -0,0 +1,139 @@
+package com.wavefront.sdk.common.clients;
+
+import com.wavefront.sdk.common.Pair;
+import com.wavefront.sdk.common.WavefrontSender;
+import com.wavefront.sdk.common.annotation.Nullable;
+import com.wavefront.sdk.entities.histograms.HistogramGranularity;
+import com.wavefront.sdk.entities.tracing.SpanLog;
+import com.wavefront.sdk.proxy.WavefrontProxyClient;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.logging.Logger;
+
+/**
+ * WavefrontMultiClient supports multiple endpoints for either Proxy or Direct Ingestion.
```

Comment:
nit: update the description to make it more clear that we replay/multicast the data onto all endpoints.